python/src/spark_rapids_ml/classification.py

```diff
@@ -985,11 +1013,21 @@ def _single_fit(init_parameters: Dict[str, Any]) -> Dict[str, Any]:
                     pdesc.rank,
                 )
 
+                intercept_array = logistic_regression.intercept_
+                # follow Spark to center the intercepts for multinomial classification
+                if (
+                    init_parameters["fit_intercept"] is True
+                    and len(intercept_array) > 1
+                ):
+                    intercept_mean = sum(intercept_array) / len(intercept_array)
```

Comment:
Is there a 'mean' method that can be called?   Also, how does this not change the model output?

Reply:
Yeah, revised the code to use np.mean and cp.mean